xpk.py

```diff
@@ -1122,12 +1175,33 @@ def make_tmp_files(per_command_name):
   Returns:
     A list of temporary files for each command.
   """
+  # Supports removal of spaces from command names before converting to file name.
   return [
-      tempfile.NamedTemporaryFile(delete=False, prefix=command + '-')
+      tempfile.NamedTemporaryFile(delete=False, prefix=command.replace(" ", "-") + '-')
       for command in per_command_name
   ]
 
 
+def get_value_from_map(key: str, map_to_search: dict) -> tuple[int, str | None]:
```

Comment:
Cool I moved to use this instead of if key ... in logic.

I like this function since it pulls the common print statement out of each request to get_value from a map.